pkg/restore/restic_restore_action.go

```diff
@@ -106,14 +106,22 @@ func (a *ResticRestoreAction) Execute(input *velero.RestoreItemActionExecuteInpu
 
 	cpuRequest, memRequest := getResourceRequests(log, config)
 	cpuLimit, memLimit := getResourceLimits(log, config)
+	if cpuRequest == "" {
```

Comment:
Can you please update the `Copyright` boilerplate on this file?

Reply:
Add 2020 onto this file?

Reply:
Yes, please.